htdocs/compta/bank/bankentries_list.php

```diff
@@ -1091,15 +1101,7 @@
 					print '</td>';
 				}
 
-				print '<td class="center">';
-				print '<input type="checkbox" id="selectAll" title="'.dol_escape_htmltag($langs->trans("SelectAll")).'" />';
-				print ' <script type="text/javascript">
-						$("input#selectAll").change(function() {
-							$("input[type=checkbox][name^=rowid]").prop("checked", $(this).is(":checked"));
-						});
-						</script>';
-				print '</td>';
-				print '<td colspan="'.($tmpnbfieldafterbalance + 2).'">';
+				print '<td colspan="'.($tmpnbfieldafterbalance + 1).'">';
```

Comment:
2 td You replaced 2 td into 1 but the second one
<td colspan="'.($tmpnbfieldafterbalance + 2).'">';
become
<td colspan="'.($tmpnbfieldafterbalance + 1).'">';

May be it should be  
<td colspan="'.($tmpnbfieldafterbalance + 3).'">';   ?

Reply:
Done